posix.mak

```diff
@@ -434,3 +434,13 @@ ${STABLE_DMD}:
 
 ${DUB}: ${DUB_DIR}/.cloned ${STABLE_DMD}
 	cd ${DUB_DIR} && DC=$(abspath ${STABLE_DMD}) ./build.sh
+
+################################################################################
+# Dman tags
+################################################################################
+
+chmgen : chmgen.d $(DMD)
+	$(DMD) -I${PHOBOS_DIR} -g chmgen.d
+
+d.tag : chmgen $(ALL_FILES)
```

Comment:
Well, strictly speaking it doesn't care, as it will follow links recursively. But the CHM probably shouldn't contain prerelease documentation, hmm.